glue/clients/ds9norm.py

```diff
@@ -123,8 +123,8 @@ def __init__(self):
         self.stretch = 'linear'
         self.bias = 0.5
         self.contrast = 1.0
-        self.clip_lo = 5.
-        self.clip_hi = 95.
+        self.clip_lo = 0.
+        self.clip_hi = 100.
```

Comment:
Unfortunately IIRC these limits don't look nice on the W5 image we use in the getting started tutorial (or I would think many IR / optical images containing both stars and ISM). I don't know if there's a smarter way to choose the defaults based on the image contents?